providers/base/api/metrics/api_query_handler.go

```diff
@@ -26,7 +27,7 @@ const (
 type APIMetrics interface {
 	// AddProviderResponse increments the number of ticks with a fully successful provider update.
 	// This increments the number of responses by provider, id (i.e. currency pair), and status.
-	AddProviderResponse(providerName, id string, status Status)
+	AddProviderResponse(providerName, id string, errorCode providertypes.ErrorCode)
```

Comment:
do we use the `Status` type anywhere now? 

is it safe / prudent to remove?

Reply:
yep!